### ISSUE TITLE
debs for binary tar downloads (e.g. netatalk, unar, etc)?

### ISSUE THREAD

**Issue by IvanExpert** [RasppleII/a2server#33]
it's what the cool debian kids like to do.

**Comment by knghtbrd** [comment]
I don't believe unar is packaged in wheezy, but it is packaged in jessie.  Netatalk is available in both, but I don't know immediately what actual source-level patches you've applied.  Looks like all the patches happen after compiling and related to stuff in the filesystem currently.  If that's the case, we can just use Debian's packages for netatalk.  If we need our own, I'll just take theirs and add patches.

**Comment by IvanExpert** [comment]
Any existing Netatalk package might be a problem. Netatalk 3.x, which is what's current and has been for some time, excised AppleTalk support, so that's out; and it's been a long time since I've looked at the 2.x package, but I remember it being unsuitable for multiple reasons. Off the top of my head, that included: a2boot missing (and therefore also not enabled via its compile option); randnum authentication missing; config files either not what I expected or not where I expected them; possibly DDP not enabled; possibly other stuff. Further, there are bug fixes that a2server-3-sharing applies to the source before compiling.

**Comment by knghtbrd** [comment]
Ahh, that's important to know.  We'll have to grab the last netatalk2 
package then and compare it to the netatalk package currently in 
Debian.  Then we package up netatalk2 which conflicts with netatalk3 
since I assume they cannot be installed at the same time given that 
they're going to want to use the same ports for afpovertcp.

Alternatively I suppose we could figure out how to modify netatalk2 
NOT to conflict with netatalk requiring both if you want to serve 
both DDP and TCP clients, but three (four really) file server daemons 
is a bit much to ask of Pi v1, and no picnic on Pi v2.

**Comment by IvanExpert** [comment]
Yeah, I think making Netatalk 2 and 3 able to operate simultaneously sounds like a diminishing rewards proposition, and Netatalk 3 will have to be removed if present. I'm sure there's stuff in Netatalk 3 that's swell and all, like Time Machine support, but if the user really wants both that and Apple II file sharing, Pi's aren't that expensive.

I'd recommend standardizing on Netatalk 2.2.4 rather than 2.2.5, which was the last 2.x release. 2.2.5 won't even compile out of the box on Debian, and was reputed to have other issues. It seemed like sort of a surprise release after they were well into 3.x. A2SERVER currently uses 2.2.4.

**Comment by knghtbrd** [comment]
bd3cf7a adds support for checking for a packaged unar.  Jessie's got one (1.8.1-3) and if we backport the package to wheezy, it'll get used automatically.

**Comment by IvanExpert** [comment]
For whatever reason, a broken unar is packaged in Raspbian Wheezy, though I don't think Debian wheezy.

I don't know what "if we backport the package to wheezy, it'll get used automatically" means; for the moment, I've made the script install unar from the package if on Jessie, and download the binary tarball if not.

**Comment by knghtbrd** [comment]
Works for me!

Joseph

**Comment by IvanExpert** [comment]
This is implemented as described (our build for Wheezy, package install if Jessie) as of 1.2.8 (maybe before).
